index.js

```diff
@@ -13,6 +13,16 @@ CleanCSSMinifier.prototype.type = 'stylesheet';
 
 CleanCSSMinifier.prototype.optimize = function(data, path, callback) {
   var error, optimized;
+
+  try {
```

Comment:
I don't think this really needs to be in a try/catch. Is that how it was done in the uglify plugin? Seems you could just validate that if `this.options.ignored` is provided it has to be a regex.